### ISSUE TITLE
MAF snps_removed is wrong

### ISSUE THREAD

**Issue by sheshen22** [suchestoncampbelllab/survivR#36]
Thank you to the developer for creating this software.
When the MAF is set, it can accurately delete SNPs. While this does not affect the outcomes in the main file, the snps_removed file that is generated is incorrect.
Using plinkCoxSurv, Figure 1 shows 3 SNPs as an example, and Figures 2 and 3 are for MAF 0.4. It is evident that the snps_removed file is incorrect.
The same happens when I analyze using my own files.
![image](https://github.com/suchestoncampbelllab/gwasurvivr/assets/104003939/e19748ea-ca6f-4634-b821-c896609eb750)
![image](https://github.com/suchestoncampbelllab/gwasurvivr/assets/104003939/62b86202-cf6a-45a3-beb3-f4e97e15871f)
![image](https://github.com/suchestoncampbelllab/gwasurvivr/assets/104003939/f0973604-eb47-452b-aec8-eb327e730700)

**Comment by aarizvi** [comment]
Indeed, this is a bug. I'm not actively maintaining this right now do plan on fixing this. 

@AliSajid if you have any time could you take a peak at this?

**Comment by AliSajid** [comment]
@aarizvi I'll take a crack at it. Self-assigning.

**Comment by aarizvi** [comment]
Fixed in 2.0.0 — the .snps_removed file now lists the correct SNPs with their real MAF values.
